apps/condo/domains/meter/schema/RegisterMetersReadingsService.js

```diff
@@ -405,6 +407,7 @@ const RegisterMetersReadingsService = new GQLCustomSchema('RegisterMetersReading
                                 commissioningDate: toISO(get(reading, ['meterMeta', 'commissioningDate'])),
                                 sealingDate: toISO(get(reading, ['meterMeta', 'sealingDate'])),
                                 controlReadingsDate: rawControlReadingsDate ? toISO(rawControlReadingsDate) : dayjs().toISOString(),
+                                isAutomatic: get(reading, ['meterMeta', 'isAutomatic']),
```

Comment:
What if value is not provided?

Reply:
Same behavior as with place `field` - ignoring. Improved a test for this case.